app/controllers/register_service_controller.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   submitRegistration: (req, res) => {
     const correlationId = req.correlationId
     const email = req.body['email']
-    const telephoneNumber = req.body['telephone-number']
+    const telephoneNumber = req.body['telephone-number'].replace(/\s/g, '')
```

Comment:
There is no trim() function for this? Or something similar? 😕

Reply:
`trim()` just removes whitespace from either end, this removes all the whitespace